app/safe/internal/server/route/secret/secret.go

```diff
@@ -96,13 +96,13 @@ func Secret(cid string, w http.ResponseWriter, r *http.Request, spiffeid string)
 		namespaces = []string{"default"}
 	}
 
-	log.DebugLn(&cid, "Secret:Upsert: ", "workloadId:", workloadId,
+	log.DebugLn(&cid, "Secret:Upsert: ", "workloadIds:", workloadIds,
```

Comment:
I’m not sure if this prints an object reference, or it’s clever enough to deserialize the array.